src/gsy_e/models/strategy/state/heatpump_tank_states/pcm_tank_state.py

```diff
@@ -126,18 +122,20 @@ def _get_condenser_temp_from_heat_demand_kWh(
         return condenser_temp_C
 
     def _limit_condenser_temp(self, condenser_temp_C: float) -> float:
-        if condenser_temp_C < self._params.min_temp_C:
-            log.warning(
-                "The PCM storage tank reached it's minimum, discharging "
+        if (self._params.min_temp_C - condenser_temp_C) > FLOATING_POINT_TOLERANCE:
+            log.error(
```

Comment:
I think that since this is not an error that will lead in an assertion or crash, it is better to stay at warning level